assets/fonts/icons/demo.html

```diff
@@ -1,1144 +0,0 @@
-<!doctype html>
```

Comment:
Are the new icons wired in to the storybook? This was a useful page before to check the icons